### ISSUE TITLE
Allow selecting Alternate Name when adding movies

### ISSUE THREAD

**Issue by schmittyd** [Radarr/Radarr#1447]
**Description:**

When adding a movie to Radarr, allow the user to select an alternative title instead of using the main US title. Also, allow the user to edit an already added movie to select an alternative title.

Check first that your problem is not listed in our wiki section:
* https://github.com/Radarr/Radarr/wiki/Common-Problems
* https://github.com/Radarr/Radarr/wiki/FAQ

Provide a description of the feature request or bug here, the more details the better. 
Please also try to include the following if you are reporting a bug

**Radarr Version:**
0.2.0.654

**Logs:**

Please use the search bar and make sure you are not submitting an already submitted issue.
Visit our [Discord server](https://discord.gg/NWYch8M) for support or longer discussions.

**Comment by galli-leo** [comment]
What would that accomplish exactly? We already store the alternative titles and use them when checking whether a found movie matches the one you want. The only thing where we could improve, is also searching with the alternative title if your indexer does not support searching by imdbid. For that we already have a feature request open.

**Comment by schmittyd** [comment]
Sorry if I wasn't clear. I meant, instead of using the original title when adding to Radarr, give the option of using the original title or using an alternative, and using that to name the title in the file system. I am from Australia, and find it annoying that it uses the main/US title, both in Radarr and the file system.

**Comment by galli-leo** [comment]
Gotcha.

**Comment by nemchik** [comment]
I would like to second this request, but I don't even think it's about a regional difference, just a user preference. For example; If I'm trying to find `Star Wars: Episode IV - A New Hope` I will only find it under the title `Star Wars`, and similarly `The Empire Strikes Back` and `Return of the Jedi` are not found by searching `Star Wars`. The end result is these files, when downloaded through Radarr, find their way to my media folder and into Plex, where I then have to setup the alternate names in Plex as well because it's much easier to find a Star Wars movie in plex by going to the S section and finding all `Star Wars: Episode # - [rest of title]` names than it is to look for `The Empire Strikes Back` which is nowhere near the majority of the other episodes. Also `Star Wars: The Force Awakens` with it's default title gives no indication as to it's chronology in the series. Star Wars is just one example, surely more can be discussed if needed, but hopefully this makes the point that many of us would just prefer to organize things (on the file system and elsewhere) using the alternate names for certain movies.

**Comment by galli-leo** [comment]
@nemchik I think you should update the titles on TMDB for such series. Maybe also hit up Travis Bell (the creator of TMDB) about keeping titles consistent. That makes way more sense, since then the titles are also consistent for users of TMDB services other than Radarr.

**Comment by nemchik** [comment]
Fair point. I'll look into that, thanks! (Didn't know I could!)

**Comment by galli-leo** [comment]
@nemchik That's the beauty of TMDB :)

**Comment by ngeraedts** [comment]
Would it be possible to re-open this issue? It seems as though the moderators at TMDB have decided to keep the short names for many movies and expect people to search alternatives via "alternative titles".

See:
https://www.themoviedb.org/talk/568eac2b9251416b4000328a

**Comment by galli-leo** [comment]
@ngeraedts This issue is already open. Also probably not gonna happen anytime soon, since we only get the alternative titles after we already added the movie.

**Comment by nemchik** [comment]
I don't normally like to point to other software and say "but X can do it!" Because the usual response is "then go use X" but I'm going to say it: couch potato.

I use Radarr because I find it much more pleasant than couch potato in every respect except this one. Radarr does an amazing job at everything it does, but it doesn't do this. We're relying on an outside source and it's making this one small part of the overall experience just a little bit worse.

**Comment by galli-leo** [comment]
@nemchik Unfortunately, we can do nothing about this, since we only get the alternative titles after we add the movie to the sql database. And as far as I can see, the issue is only with the first three Star Wars movies? So you can probably rename the folders manually for the time being.

**Comment by ngeraedts** [comment]
Sorry for incorrectly asking of the reopen - I saw the referenced issue and my tired brain misinterpreted.

I guess manual intervention with "Static Path: Yes" will do for the time being.

**Comment by nemchik** [comment]
[AVP: Alien vs. Predator (2004)](https://www.themoviedb.org/movie/395-avp-alien-vs-predator)
[Aliens vs Predator: Requiem (2007)](https://www.themoviedb.org/movie/440-aliens-vs-predator-requiem)
These are pretty inconsistent in their formatting.

The Captain America movies lack chronology (their titles don't indicate what order they should be in).

The dark knight trilogy starts with Batman begins and again none of the 3 indicate chronology in their titles.

The divergent series.

The fast and the furious series.

Harry Potter.

Hellraiser.

The Hobbit.

The second Jurassic Park moved is called "The Lost World", and again the whole series has no chronology in the names.

Kingsman.

The Lord of the rings

The matrix

Pirates of the Caribbean

The new JJ Abrams Start Trek movies

Thor

Underworld

X-Men

And I'm sure there's more.

Some of these are the fault of the creators giving their movies shitty names and making a series with names that don't match. But pretty much every example above can be solved by alternate titles.

**Comment by burgerga123** [comment]
I would also like to request this feature. I understand you only grab the alternate titles after the movie is added. It would seem simple (to me) to add a column of radio buttons on the "Titles" tab of the movie where you could select the title that Radarr uses to display, sort, and rename.

<img width="1341" alt="screenshot 2018-03-01 21 27 25" src="https://user-images.githubusercontent.com/1574828/36884748-9b563f58-1d97-11e8-9827-89f0398bde2a.png">

**Comment by LandMineHare** [comment]
This feature would be great for the James Bond movies as well. Right now they're just sitting all over the place. However being able to rename them "James Bond I: Dr. No", "James Bond II: From Russia With Love" etc would make it so much cleaner looking, IMHO.

**Comment by djmixman** [comment]
I have nothing else to add other than this feature would be nice.

**Comment by MoeJoe87** [comment]
Please i need that too...otherwise it is not/never matching some titles and the whole cut off also not works...just manual by hand...is it still not possible to pick up an alternat ename? Because of that it would already be solved. ;)

**Comment by nemchik** [comment]
From a subtitle perspective look up bazarr. It integrates with radarr and should handle subtitles for you.

**Comment by tronder88** [comment]
Yes please, possible to store as alternative title would be great!

**Comment by nipuna-perera** [comment]
+1 I've seen some movies get selected with wrong title names which I would like to edit myself

**Comment by superdonpan** [comment]
I wonder if this really is that hard to sort out? I've added movies with full Swedish titles (both via "add movie" and "Bulk import") and it still finds the movies on tMDB and matches them. So it must search alternative/original titles already before a movie is added. Problem is it still names it after the English title and I can't change it after the movies been added either. 

So in short: it seems like it's doing half of what we want it to do but just not allowing us to display the alternative/original title.

![image](https://user-images.githubusercontent.com/23033257/84580429-2e06e880-add7-11ea-85f0-024cfdec1e58.png)

![image](https://user-images.githubusercontent.com/23033257/84580456-81793680-add7-11ea-9ec7-3060a2f51a98.png)

**Comment by LeoShivas** [comment]
> I wonder if this really is that hard to sort out? I've added movies with full Swedish titles (both via "add movie" and "Bulk import") and it still finds the movies on tMDB and matches them. So it must search alternative/original titles already before a movie is added. Problem is it still names it after the English title and I can't change it after the movies been added either.
> 
> So in short: it seems like it's doing half of what we want it to do but just not allowing us to display the alternative/original title.
> 
> ![image](https://user-images.githubusercontent.com/23033257/84580429-2e06e880-add7-11ea-85f0-024cfdec1e58.png)
> 
> ![image](https://user-images.githubusercontent.com/23033257/84580456-81793680-add7-11ea-9ec7-3060a2f51a98.png)


Same example here :
![image](https://user-images.githubusercontent.com/28844182/100338621-1adc8380-2fd9-11eb-9bf8-2f4e2986fc30.png)

I'm looking for the french movie "Jusqu'ici tout va bien". This is the original name.

My language profile when adding is "French" :
![image](https://user-images.githubusercontent.com/28844182/100338866-6c850e00-2fd9-11eb-957b-43a94ddaee77.png)

But the Radarr title of the movie is US :
![image](https://user-images.githubusercontent.com/28844182/100339103-b8d04e00-2fd9-11eb-970c-382217baa2df.png)

**Comment by John-Gee** [comment]
Same as right above, when adding a non English movie, it often renames it to the English title even if that's not the one I gave and tmdb has the other one. It'd be good to be able to manually change that, and better to be able to auto pick the title from Language L, as part of the bulk rename or refresh.

**Comment by Sergiodiaz53** [comment]
We have the same problem with Spanish language. I just saw one of my trackers uploaded a film called 'La isla de los bucaneros'. When I add it in Radarr it does it with its English name (''Against All Flags''), then I go to the manual search tab and I can only find movies called ''Against All Flags'' in my non-spanish trackers, but no spanish result. 

Of course I have all my languages profiles specifically in Spanish.

**Comment by John-Gee** [comment]
So I ended up working around the wrong filenames with a script (when I add a movie I give it the correct folder name and then I force matching between folder and files, and then the nfo's content too). We could also script renaming the title in radarr, but I'm not sure if that'd be safe.

**Comment by John-Gee** [comment]
Well it won't work like that. From a quick look at the DB, my feeling is that Radarr will display the title from table MovieTranslations that has the smallest value for the column Language, with English being at 1 and I think Spanish is at 3. Potentially, assuming there is a row with Language at 3 you could just delete the previous rows for that movie and that would work. Actually if we could get a list of MovieId per languages that'd be easy, so I guess get all Movies that use certain ProfileId then look at their MovieTranslations row and keep only the row you like. I'm afraid that might reduce the pool for searching though, so I don't know if that'd be a good work around.

**Comment by amazondevil** [comment]
Hi All,
I'm new to Radarr, and I too am looking for a feature or function to, at the very least, select one of the alternative titles.  I can see the list of alternative names but unlike one of the screenshots in the thread above, there does not appear to be any way to select them.
![Screenshot 2021-03-16 002507](https://user-images.githubusercontent.com/80579633/111160356-1d95cf80-85ee-11eb-8bd8-85415770267e.png)

**Comment by bakerboy448** [comment]
@amazondevil 
Please use one of the support channels: [Reddit](https://www.reddit.com/r/radarr/) or  [Discord](https://radarr.video/discord) for support/questions.

This is a feature request that has not been implemented yet

There are _some_ UI options available.
https://wiki.servarr.com/Radarr_Settings#Language

**Comment by elasticwax** [comment]
+1 for being able to select a specific alternative title. Another use case also deals with Star Wars. I have multiple versions of Episode IV: A New Hope because George Lucas decided to release a few versions. Radarr sees them all as Star Wars but I want to be able to specify which one is the Despecialized version and which one is the most recent release with the CGI added.

**Comment by peloyeje** [comment]
Hi! +1 for this feature as well, would be very useful for French people like myself that prefer original titles for pretty much everything except for certain movies (one good example lately would be Parasite) :)
I see that this feature request is planned for the current version, what's the expected timeframe for this development? Is there anything I can do to help? I'm not proficient in C# but would happily contribute to a bounty or whatever is the most appropriate

**Comment by charredchar** [comment]
I can't believe how long this has been here... But +1! I was searching for a method to do this for the exact Star Wars example given before. I agree with the [option to use button in the alt-name tab](https://github.com/Radarr/Radarr/issues/1447#issuecomment-369824617) just so the display and file names match for better library sorting. My media programs (Jellyfin, Kodi) will use the metadata for naming so this makes their movie lists very messy and unpleasant as well.

**Comment by Marvellll** [comment]
+1
selecting an alternative title would be great

**Comment by KevoM** [comment]
Could alternate titles or desired language titles be attributed to the sort title tag ? (<sorttitle> in .nfo)
The way Radarr treats name and translated names is confusing at the least.

If you set your radarr in a language, the default name displayed (UI) is english no matter what. 
Fortunately, you can format it to the desire language via {Movie.Title:country} (import files)
But the sort title tag (.nfo) will be exclusively be in english...

<img width="665" alt="Capture d’écran 2022-08-15 à 17 51 17" src="https://user-images.githubusercontent.com/8274366/184669943-d941790c-6e58-4419-98d4-27bc6d1cc23e.png">

Leading to this kind of unsorted library:
<img width="1652" alt="Capture d’écran 2022-08-15 à 17 48 24" src="https://user-images.githubusercontent.com/8274366/184671627-69d8e435-b5e4-4f8d-af6e-93e7a6a7d0d7.png">

Because of this sort title : 
<img width="924" alt="Capture d’écran 2022-08-15 à 17 48 54" src="https://user-images.githubusercontent.com/8274366/184671688-8594ca83-4b90-4279-8221-318c62d355e4.png">

**Comment by rui-nar** [comment]
+1 everything works except showing the right title (IE, the original name)

**Comment by planetrocky** [comment]
+1 Needs to be a way to select alternate title

**Comment by deedaleet** [comment]
+1 this would be a great feature.

**Comment by hendawyjr** [comment]
_After having experienced a sudden leap through time from the year 2017 to the present day, I have come to the conclusion that numerous individuals are expressing their desire for this minor functionality._ 

**Nevertheless, no one has yet taken it upon themselves to create or discover an alternative solution.** 

@RobinDadswell It is presumed that such a feature would predominantly facilitate the augmentation of language capabilities, particularly for distinguished languages like Arabic, Hebrew, Spanish and French. In essence, this expansion contributes to broadening one's global imprint. - Regards, Tehuti

**Comment by ajamico** [comment]
+1, would definitely be helpful.

**Comment by letraitplat** [comment]
+1 in here as well
TBH I thought this feature was already there because so useful and 101 (it has impacts on your overall downloading - managing - browsing for content to view experience) and that I was the one not using / setting up radarr properly, but, for once, issue is not on my end :)

That would be so beneficial for so many users... !

**Comment by bakerboy448** [comment]
A simple `+1` react on the OP would suffice.

additional comments indicating me too or I want this are largely noise and rather spammy.

**Comment by TheBozzer** [comment]
+1 from me..........Would help with my OCD :)

**Comment by emhl** [comment]
+1 i don't want to manually rename everything afterwards

**Comment by bakerboy448** [comment]
A simple +1 on the OP would suffice

Comments that are not productive to the issue at hand such as +1, me too, i want this too, when will this be in, any updates, etc. are poor etitique and do not result in issues being worked on any faster. Repeated comments of this nature will result in the thread being locked.

**Comment by Linguista** [comment]
This feature would also be extremely useful for those of us who watch non-English films. I have about 900 German-language films, and for some reason about 1/3 of them have their _English-language_ titles in Radarr, in spite of the fact that I _always_ search for them and add them to Radarr using the original German titles. Perhaps this is a bug in and of itself.

But regardless, this behavior means that I can't search for a third of my German films, because their titles are in English both in Radarr and on the disk (thanks to Radarr's folder creation and file renaming using the wrong language title). 

I'm prepared to manually fix this problem for each film, but without the requested feature it's impossible.

**Comment by bakerboy448** [comment]
# A simple +1 on the OP would suffice

# Comments that are not productive to the issue at hand such as +1, me too, i want this too, when will this be in, any updates, etc. are poor etitique and do not result in issues being worked on any faster. Repeated comments of this nature will result in the thread being locked.

Please use [Discord](https://radarr.video/discord) for support/questions.

**Comment by Linguista** [comment]
> # A simple +1 on the OP would suffice
> # Comments that are not productive to the issue at hand such as +1, me too, i want this too, when will this be in, any updates, etc. are poor etitique and do not result in issues being worked on any faster. Repeated comments of this nature will result in the thread being locked.
> Please use [Discord](https://radarr.video/discord) for support/questions.

First of all, there is absolutely no reason for you to be rude. _That_ is poor etiquette.

Secondly, don't insult me by brushing off a detailed comment containing specific examples, some of which are new information on this issue, as if it were a mere "+1", etc. If you think that's what it is, you either didn't understand my comment or didn't bother to read it all the way through. 

Thirdly, even if my comment _were_ a "+1" or similar, that would be entirely warranted on an issue that has been open for **7 years** and has seen no actual information from a dev in **2 years** (your copypasting of admonitions to shut up about this open issue don't count as information, obviously).

Finally, threatening to lock a thread if people use it a rather bizarre and self-defeating threat. But you might as well, as driving away good-faith users who care enough about the software to find, go to and participate in its issue tracker or subreddit seems to be a veritable Radarr tradition. Plus,after seven years you might as well mark this issue as `DONT-KNOW-HOW-TO-FIX` and lock it.

**Comment by bakerboy448** [comment]
The main development team is split across 5+ apps: Lidarr, Prowlarr, Radarr, Readarr, Whisparr, Sonarr upstream commits
There are NO ACTIVE developers who work on this, and exactly zero work on this full-time. Exactly zero of any members involved (developers / support) get paid to do this. Everyone of them have full-time real jobs / kids and family / more than just this project.

As you can imagine between the projects (in addition to the various backends and other modules), the todo list is long and time short.

Clearly no one from the community believes this is an issue and thus no one from the community has worked on this issue.

**Comment by Linguista** [comment]
Your utter lack of self-awareness is amusing. You state that "no one from the community believes this is an issue" while at the same time attacking and threatening anyone who dares to make it an issue, or a bigger issue, by commenting in this thread.

This is what is known as a self-fulfilling prophecy.

**Comment by bakerboy448** [comment]
If you - or anyone on this thread - believe this is so easy to implement and that the community believes this is a critical missing feature that's needed then looking forward to the pull request....so far over 7 years no one can be bothered.

Please immediately cease your spam and personal attacks.

**Comment by solbu** [comment]
> Clearly no one from the community believes this is an issue and thus no one from the community has worked on this issue...instead the community can only complain as to why no one has worked for free on this issue.

I disagree. Clearly Some of the community cares. It might not be a very big part, but some do.

A more productive way forward would be for those that care and which have the skills to implement it, would do so and send a Pull Request.
I'm sure that if this happened, the team would at some point test the PR. And if all the checks passes and it is within the quality the team wants – maybe it will be merged.

**Comment by John-Gee** [comment]
> Your utter lack of self-awareness is amusing. You state that "no one from the community believes this is an issue" while at the same time attacking and threatening anyone who dares to make it an issue, or a bigger issue, by commenting in this thread.
> 
He's not wrong though, we all dislike the lack of this feature, but none of us has done more than complain about it, ergo we must not dislike it that badly. If @bakerboy448 is the one triaging reports, then it just wastes his time to get notifications instead of a +1 he would not look at.

I wonder if I can write a quick script to hack around that, I've done it for similar stuff in the past, it may not be that hard.

**Comment by Chacsam** [comment]
+1
